CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [X.X.X]
+
+`2021-XX-XX`
+
+### What's new
+
+- **Added optional tooltip for `Wizard` steps.**
```

Comment:
Maybe mention that is `description` property in `StepProperties`.

Reply:
Adjusted changelog.